crosswalk/index.html

```diff
@@ -276,6 +276,31 @@ <h2>EPUB and ONIX</h2>
 								Specification 1.0 AA</p>
 						</td>
 					</tr>
+                    
+					<tr>
+						<td>
+							<p><code>a11y:exemption</code> with the text string<br/><code>microenterprise</code></p>
```

Comment:
The values look like the ONIX descriptions. The [three epub codes](https://www.w3.org/TR/epub-a11y-exemption/#exemption-values) you use with the property are:

- eaa-microenterprise
- eaa-disproportionate-burden
- eaa-fundamental-alteration